src/components/search-bar.component.ios.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component, PropTypes } from 'react';
+import RNSearchBar from 'react-native-search-bar';
```

Comment:
I have the same question, since we're using `react-native-elements` I think we should stick with that instead of start adding more deps into the project. You can find the component of `react-native-elements` in here react-native-search-bar